src/luaotfload-database.lua

```diff
@@ -235,11 +239,9 @@ end
 
 --- string -> string
 
-local invalidchars = "[^%a%d]"
-
 local sanitize_fontname = function (str)
     if str ~= nil then
-        str = utf8gsub (utf8lower (str), invalidchars, "")
```

Comment:
This didn't actually work: For example the "sanitized" form of "Häää?" was "H���", that is a "H" followed by three invalid UTF-8 sequences. Also it requires slnunicode.